### ISSUE TITLE
RPC "eth_getBalance" does not work with MEW

### ISSUE THREAD

**Issue by thanhson1085** [BuildOnViction/tomochain-v1#389]
User reported that they can not use MEW with TomoChain. I tested it, It seems relate to this logs:
```
ERROR[01-06|05:22:37] RPC serveRequest goroutine 70220605 [running]:
ERROR[01-06|05:22:35] RPC serveRequest goroutine 19504605 [running]:
ERROR[01-06|05:22:34] RPC serveRequest goroutine 19504558 [running]:
ERROR[01-06|05:22:34] RPC serveRequest goroutine 70219662 [running]:
ERROR[01-06|05:22:34] RPC serveRequest goroutine 70219647 [running]:
```

Request from MEW:
```
curl 'https://rpc.tomochain.com/' -H 'Accept: application/json, text/plain, */*' -H 'Origin: https://www.myetherwallet.com' -H 'User-Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_14_2) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/71.0.3578.98 Safari/537.36' -H 'Content-Type: application/json; charset=UTF-8' --data-binary '{"method":"eth_getBalance","params":["0x44dba649142b131021aac7c5608c0cd123f5c58e","pending"],"id":"e083a7d85857a03d4634e23d919c9ba7","jsonrpc":"2.0"}' --compressed
```

**Comment by thanhson1085** [comment]
This is OK:
```
curl 'https://rpc.tomochain.com/' -H 'Accept: application/json, text/plain, */*' -H 'Origin: https://www.myetherwallet.com' -H 'User-Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_14_2) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/71.0.3578.98 Safari/537.36' -H 'Content-Type: application/json; charset=UTF-8' --data-binary '{"method":"eth_getBalance","params":["0x44dba649142b131021aac7c5608c0cd123f5c58e", "latest"],"id":"e083a7d85857a03d4634e23d919c9ba7","jsonrpc":"2.0"}' --compressed
{"jsonrpc":"2.0","id":"e083a7d85857a03d4634e23d919c9ba7","result":"0x2373d8fe36b000"}
```

**Comment by thanhson1085** [comment]
```
> eth.getBalance("0x44dba649142b131021aac7c5608c0cd123f5c58e", "pending")
Error: EOF
    at web3.js:3143:20
    at web3.js:6347:15
    at web3.js:5081:36
    at <anonymous>:1:1
```

```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x48 pc=0xab5f29]

goroutine 34413 [running]:
github.com/ethereum/go-ethereum/miner.(*worker).pending(0xc0002d6500, 0x0, 0x0)
	/home/vagrant/go/src/github.com/ethereum/go-ethereum/build/_workspace/src/github.com/ethereum/go-ethereum/miner/worker.go:191 +0x99
```

**Comment by thanhson1085** [comment]
fixed #390
